src/interpret.cpp

```diff
@@ -2989,15 +2991,21 @@ invoke : {
   if (method->flags() & ACC_NATIVE) {
     invokeNative(t, method);
   } else {
+    if (DebugCalls && method) {
```

Comment:
Doesn't seem necessary to check that `method` is non-null here, since we dereferenced it above.